pkg/dataplane/util/ansible_execution.go

```diff
@@ -58,190 +56,191 @@ func AnsibleExecution(
 	var sshKeyMountPath string
 	var sshKeyMountSubPath string
 
+	const jobRestartPolicy string = "OnFailure"
+
 	ansibleEEMounts := storage.VolMounts{}
 
 	executionName, labels := GetAnsibleExecutionNameAndLabels(service, deployment.GetName(), nodeSet.GetName())
-	ansibleEE, err := GetAnsibleExecution(ctx, helper, deployment, labels)
+
+	existingAnsibleEE, err := GetAnsibleExecution(context.TODO(), helper, deployment, labels)
```

Comment:
Why switch to `TODO` ?

Reply:
Yeah, changed it back and added ctx back to the function signature. Thanks